__tests__/reducers/index.test.js

```diff
@@ -80,6 +80,25 @@ describe('createReducer', () => {
   })
 })
 
+describe('setPublishError', () => {
+  it('adds error to editor state', () => {
+    const newState = setPublishError(initialState.selectorReducer, {
+      type: 'RETRIEVE_ERROR',
+      payload: { resourceTemplateId: 'abc123' },
```

Comment:
why would it have a `resourceTemplateId` here? Shouldn't the payload just be a string with the reason?

Reply:
I was a bit muzzy on this.  It can certainly just have the error.

Reply:
is there a better way to test these that avoids mocking errors passing?